### PR TITLE
Add cockpit, remove httpd-admin

### PR DESCRIPTION
**Pull request by gsanchietti**
NethServer/dev#6320

### CODE REVIEW

File: nethserver-enterprise-groups.xml.in [hunk 1]
```diff
@@ -1162,6 +1162,16 @@
       </packagelist>
   </group>
 
+  <group>
+    <id>nethserver-httpd-admin</id>
+    <_name>Old Server Manager</_name>
```
Comment (DavidePrincipi) [inline nethserver-enterprise-groups.xml.in:1167]:
```suggestion
    <_name>Nethgui Server Manager</_name>
```

File: po/comps.pot [hunk 2]
```diff
@@ -506,37 +506,45 @@ msgid "Scan mail attachments with Yomi sandbox"
 msgstr ""
 
 #: ../nethserver-enterprise-groups.xml.in.h:122
-msgid "Base system"
+msgid "Old Server Manager"
```
Comment (DavidePrincipi) [inline po/comps.pot:509]:
```suggestion
msgid "Nethgui Server Manager"
```

File: nethserver-enterprise-groups.xml.in [hunk 3]
```diff
@@ -1162,6 +1162,16 @@
       </packagelist>
   </group>
 
+  <group>
+    <id>nethserver-httpd-admin</id>
+    <_name>Nethgui Server Manager</_name>
+    <_description>Legacy Server Manager based on NethGUI</_description>
```
Comment (DavidePrincipi) [inline nethserver-enterprise-groups.xml.in:1168]:
```suggestion
    <_description>Legacy Server Manager web application on HTTPS port 980</_description>
```

File: po/comps.pot [hunk 4]
```diff
@@ -506,37 +506,45 @@ msgid "Scan mail attachments with Yomi sandbox"
 msgstr ""
 
 #: ../nethserver-enterprise-groups.xml.in.h:122
-msgid "Base system"
+msgid "Nethgui Server Manager"
 msgstr ""
 
 #: ../nethserver-enterprise-groups.xml.in.h:123
-msgid "Common packages to all products"
+msgid "Legacy Server Manager based on NethGUI"
```
Comment (DavidePrincipi) [inline po/comps.pot:513]:
```suggestion
msgid "Legacy Server Manager web application on HTTPS port 980"
```

**Review by DavidePrincipi — APPROVED** [review]

**Comment by gsanchietti** [comment]
I'm not fond of such changes @DavidePrincipi 
Inside the doc we always used "Old Server Manager" vs "New Server Manager".
We never used "NethGUI Server Manager"

**Pull request merged**